<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckSumScannerLog</class>
 <widget class="QDialog" name="CheckSumScannerLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check-sum scanner log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="plainTextEditLog">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxEnableLog">
       <property name="toolTip">
        <string>Enable log output</string>
       </property>
       <property name="text">
        <string>Enable log</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMaxLogSize">
       <property name="text">
        <string>Log size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxLogSize">
       <property name="toolTip">
        <string>Maximum number of lines in log (0 = no limit)</string>
       </property>
       <property name="specialValueText">
        <string>unlimited</string>
       </property>
       <property name="suffix">
        <string> lines</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Scanner progress indicator</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>Idle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear log</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Close this window</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>plainTextEditLog</tabstop>
  <tabstop>checkBoxEnableLog</tabstop>
  <tabstop>spinBoxMaxLogSize</tabstop>
  <tabstop>toolButtonClear</tabstop>
  <tabstop>toolButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>CheckSumScannerLog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEditLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
